go/logic/migrator.go

```diff
@@ -734,6 +745,13 @@ func (this *Migrator) initiateServer() (err error) {
 	return nil
 }
 
+// attemptInstantDDL tries to apply the DDL statement directly to the table
```

Comment:
Last nit, I swear 😅! This func (mostly the comment) feels redundant

What do you think about `.Migrate()` calling `this.applier.AttemptInstantDDL()` directly? I think that might be more straightforward to read, but totally optional